crates/host/src/wasmbus/mod.rs

```diff
@@ -952,7 +952,7 @@ impl KeyValueReadWrite for Handler {
             .call_operation(
                 targets.get(&TargetInterface::WasiKeyvalueReadwrite),
                 METHOD,
-                &key,
+                &wasmcloud_compat::keyvalue::DeleteRequest { key },
```

Comment:
Doing this would break backwards-compatiblity, since the Smithy-defined RPC expects a plain string here, we cannot change the format here https://github.com/wasmCloud/interfaces/blob/f020c93d4cacd50318301f686e2f059a15862e1e/keyvalue/keyvalue.smithy#L81

Reply:
This is known -- this commit was made in order to get it working with the default methodology of bindgen as of right now.